01-module-system/helper.js

```diff
@@ -0,0 +1,59 @@
+const fs = require('fs')
+const chalk = require('chalk');
+function getNotes () {
+    try {
+        const data = fs.readFileSync('notes.json')
+        const content = data.toString()
+        const jsonData = JSON.parse(content)
+        return jsonData
+    }catch(err){
+        console.log(chalk.red("err"))
+       return []
+    }
+    }
```

Comment:
fix indentation: console.log and return should have same indentation as  on lines 4,5,6,7. Also lines 13 curly don't need this tab